spec/models/event_spec.rb

```diff
@@ -1,8 +1,136 @@
 require 'spec_helper'
 
 describe Event do
+  let(:event) { create(:event) }
+  let(:user) { create(:user) }
 
-  describe 'abstract_word_count' do
+  describe 'validations' do
+    it 'has a valid factory' do
+      expect(build(:event)).to be_valid
+    end
+
+    it 'is not valid without a title' do
+      expect(event.title.present?).to be true
+    end
+
+    it 'is not valid without an abstract' do
+      expect(event.abstract.present?).to be true
+    end
+
+    it 'is not valid without a program' do
+      expect(event.program.present?).to be true
+    end
+
+    it 'is not valid without an event_type' do
+      expect(event.event_type.present?).to be true
+    end
+
+    context '#abstract_limit' do
+      before :each do
+        event.event_type.maximum_abstract_length = 2
+        event.event_type.minimum_abstract_length = 2
+      end
+
+      it 'returns error message if abstract is too long' do
```

Comment:
Maybe you can group this in a nested context ``valid`` and ``invalid``. That would make the test name shorter...